software/verification/verify_cdl_sequence.py

```diff
@@ -567,10 +638,14 @@ def run_cdl_simulation(self, model, output_folder, ip_list, op_list, sample_time
             err = omc.sendExpression("getErrorString()")
             print("error while loading Modelica Buildings Library: {}".format(err))
```

Comment:
Shouldn't all these errors raise an exception? Otherwise, a program that implements this script will have a very difficult time catching errors. Also, above you correctly start the exception message with a capital letter, but here you use lower case. Please use correct punctuation and capitalization throughout.